sfm/sfm/settings/common.py

```diff
@@ -125,7 +125,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 
-STATIC_URL = '/static/'
+STATIC_URL = '/ui/static/'
```

Comment:
This puts all of the static files in /ui/static/, including the admin files.  I think you  want to leave this as /static/ and then put the sfm-ui static files in sfm/ui/static/ui/ instead of sfm/ui/static/.

See https://docs.djangoproject.com/en/1.9/howto/static-files/